components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/revoke/OAuthRevocationEndpoint.java

```diff
@@ -72,23 +73,18 @@ public Response revokeAccessToken(@Context HttpServletRequest request,
                 logAccessTokenRevocationRequest(httpRequest);
             }
             String token = httpRequest.getParameter("token");
-            if (token == null || token.equals("")) {
-                if (paramMap.get("token") != null && !paramMap.isEmpty()) {
-                    token = paramMap.get("token").get(0);
-                }
+            if (StringUtils.isBlank(token) && paramMap.get("token") != null && !paramMap.isEmpty()) {
```

Comment:
why paramMap empty check for? if it is empty, value of "token" key would have been null